pandas/core/indexes/accessors.py

```diff
@@ -193,7 +193,7 @@ class DatetimeProperties(Properties):
 
     def to_pydatetime(self) -> np.ndarray:
         """
-        Return the data as an array of native Python datetime objects.
+        Return the data as an array of :class:`datetime.datetime`.
```

Comment:
could keep the word "objects" like you did in to_pytimedelta and a "date" above (but not time or timetz)?

Reply:
Sounds good. Changed.